lib/persisted-model.js

```diff
@@ -641,7 +641,7 @@ module.exports = function(registry) {
       accessType: 'WRITE',
       accepts: {
         arg: 'data', type: 'object', model: typeName,
-        description: 'Model instance data',
+        description: 'Model instance data', allowArray: true,
```

Comment:
It's a good practice to keep related things close together and unrelated things apart. IMO, `allowArray` and `description` are rather unrelated.

Please move `allowArray` closer to `type`/`model`, e.g. on the previous line.